packages/shared/apis/IEngineAPI.ts

```diff
@@ -20,11 +20,6 @@ export interface IEngineAPI {
    */
   sendBatch(actions: EntityAction[]): Promise<void>
 
-  /**
-   * Start signal, sent after everything was initialized
-   */
-  startSignal(): Promise<void>
```

Comment:
Where did this go?

Reply:
it was dead code or related to old `decentraland-rpc`. it was being used nowhere